packages/app-project/src/components/AuthModal/components/LoginForm/components/Form/Form.spec.js

```diff
@@ -1,25 +1,31 @@
 import React from 'react'
-import { shallow } from 'enzyme'
-import { expect } from 'chai'
+import { mount, shallow } from 'enzyme'
 import sinon from 'sinon'
-import { TextInput } from 'grommet'
+import { Grommet, TextInput } from 'grommet'
+import zooTheme from '@zooniverse/grommet-theme'
 import {
   userNameFieldId,
   passwordFieldId,
   Form
 } from './Form'
 
-describe('Component > Form', function () {
+describe('LoginForm > Component > Form', function () {
   it('should render without crashing', function () {
-    const wrapper = shallow(<Form />)
+    const wrapper = shallow(
+      <Form />,
+      { wrappingComponent: <Grommet />, wrappingComponentProps: { theme: zooTheme } }
+    )
     expect(wrapper).to.be.ok()
   })
 
   describe('fields', function () {
     let rendered
 
     before(function () {
-      const wrapper = shallow(<Form />)
+      const wrapper = mount(
+        <Form />,
+        { wrappingComponent: Grommet, wrappingComponentProps: { theme: zooTheme } }
```

Comment:
If you're re-using this, is it worth declaring it as a constant somewhere?

Reply:
I'm also a little unclear as to whether `wrappingComponent` should be a component or an instance of a component. You seem to be doing both in the code.

Reply:
Yep. Shallow only worked with `<Grommet />`, and mount only worked with `Grommet`. I couldn't find any explanation in the docs.

Reply:
Almost all the shallows in the tests and the single mount here have slight differences (changes in the props on `Form` itself for instance), so there's not much re-use actually.

Reply:
Or you mean the options object itself? That I can do

Reply:
Yeah, I was thinking something like
```js
const formContext = { wrappingComponent: Grommet, wrappingComponentProps: { theme: zooTheme }}
```
might save some repetition.